code-samples/command-handling/adding-features/11/commands/utility/reload.js

```diff
@@ -11,10 +13,28 @@ module.exports = {
 			return message.channel.send(`There is no command with name or alias \`${commandName}\`, ${message.author}!`);
 		}
 
-		delete require.cache[require.resolve(`./${command.name}.js`)];
+		const commandFolders = fs.readdirSync('./commands');
+
+		let folderName;
+		let folderFound = false;
+
+		for (const folder of commandFolders) {
+			const commandFiles = fs.readdirSync(`./commands/${folder}`).filter(file => file.endsWith('.js'));
+			for (const file of commandFiles) {
+				const fileName = file.slice(0, -3);
+				if (commandName === fileName) {
+					folderName = folder;
+					folderFound = true;
+					break;
+				}
+			}
+			if (folderFound) break;
+		}
```

Comment:
This could be simplified by making use of `Array#find()` along with `Array#includes()` and instead of filtering out and slicing out the `.js` extension it would make life easier to just directly look for that (`'command.js'`) among the command files within each categroy.
```suggestion
		const folderName = commandFolders.find(folder => fs.readdirSync(`./commands/${folder}`).includes(`${commandName}.js`));
```